### PR TITLE
Version-bump-v2.7.3

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
= 2.7.3 - MONDAY, 31th JULY 2023 =
* Improvement: Core Blocks - Enhanced the reusable block functionality for all Core Blocks, with the exception of the Container.
* Improvement: Global - Optimized the editor performance and stability.

- [x]  Run to update version number : `grunt bump-version --ver=[version-number]`
- [x]  Verify `Stable tag` is [version-number] in readme.txt
- [x]  Verify `Tested upto` is set to latest tested version of WordPress
- [x]  Verify changelog `date` and `content` as per Spectra standards
- [x]  Generate README.md : `grunt readme`
- [x]  Update `version` in `ultimate-addons-for-gutenberg.php` in plugin description
- [x]  Verify constant `UAGB_VER` in `class-uagb-loader.php` with latest version of Spectra
- [x]  Generate POT file : `npm run makepot`
- [x]  Update version in `package.json `and `package-lock.json`

### CODE REVIEW

**Review by AbhijitNage123 — CHANGES_REQUESTED** [review]
Pot file is missing

File: README.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
-### X.X.X - THURSDAY, 27th JULY 2023 ###
+### 2.7.3 - MONDAY, 31th JULY 2023 ###
```
Comment (bsf-zanev) [inline README.md:218]:
```suggestion
### 2.7.3 - MONDAY, 31st JULY 2023 ###
```

Comment (rajmohanreddy774) [reply README.md]: Done changes.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by bsf-zanev — APPROVED** [review]

File: readme.txt [hunk 2]
```diff
@@ -1509,3 +1506,8 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 = 0.0.1 =
 * Initial release
+
+== Upgrade Notice ==
+
+= 2.7.3 =
+We've optimized the internal structure of Spectra in this update. To ensure your safety, please backup your website before updating!.
```
Comment (bsf-zanev) [inline readme.txt:1513]:
```suggestion
We've optimized the internal structure of Spectra in this update. To ensure your safety, please backup your website before updating!
```

Comment (rajmohanreddy774) [reply readme.txt]: Done.

**Review by bsf-zanev — APPROVED** [review]
Looks good, in the future, we can add a rule to exclude upgrade notices in the Readme file if needed.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**